### PR TITLE
github: update tagging instructions in release checklists

### PR DESCRIPTION
**Pull request by marquiz**
Simplify and standardize the git tag message body. Adding the full changelog in the tag description feels unnecessary and cannot be corrected after the fact.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-patch-release.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ Please do not remove items from the checklist
   - [ ] Write the change log into the draft release
   - [ ] Upload release artefacts generated by `prepare-release.sh` script above to the draft release
 - [ ] An OWNER runs
-     `git tag -s $VERSION`
-      and inserts the changelog into the tag description.
+     `git tag -s -m"NFD release $VERSION" $VERSION`
```
Comment (ArangoGutierrez) [inline .github/ISSUE_TEMPLATE/new-patch-release.md:21]:
```suggestion
     `git tag -s -m "NFD release $VERSION" $VERSION`
```

File: .github/ISSUE_TEMPLATE/new-release.md [hunk 2]
```diff
@@ -27,8 +27,7 @@ Please do not remove items from the checklist
   - [ ] Write the change log into the draft release
   - [ ] Upload release artefacts generated by `prepare-release.sh` script above to the draft release
 - [ ] An OWNER runs
-     `git tag -s $VERSION`
-      and inserts the changelog into the tag description.
+     `git tag -s -m"NFD release $VERSION" $VERSION`
```
Comment (ArangoGutierrez) [inline .github/ISSUE_TEMPLATE/new-release.md:30]:
```suggestion
     `git tag -s -m "NFD release $VERSION" $VERSION`
```

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]
A space is needed no?

File: .github/ISSUE_TEMPLATE/new-patch-release.md (hunk 1 above)
Comment (marquiz) [reply .github/ISSUE_TEMPLATE/new-patch-release.md]: Yeah, maybe this is cleaner. Changed as suggested

File: .github/ISSUE_TEMPLATE/new-release.md (hunk 2 above)
Comment (marquiz) [reply .github/ISSUE_TEMPLATE/new-release.md]: Changed as suggested

**Comment by marquiz** [comment]
No space is needed but maybe it looks better as you suggested

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**